Assets/Scripts/Player/HandleSpriteRotation.cs

```diff
@@ -8,15 +8,13 @@ namespace Player
 {
     public class HandleSpriteRotation : MonoBehaviour
     {
-        [SerializeField] private float _startRotationDistance;
-        [SerializeField] private float _defaultRotationSpeed;
+        [SerializeField] private float _startRotationDistance = 5f;
```

Comment:
Not sure about that. What does it mean? Are we suppose to take this value or inspector one?